features/support/testing_dsl.rb

```diff
@@ -98,6 +98,10 @@ def project_dir
         Paths.project(name)
       end
 
+      def cloneGoProject(fixture_name)
```

Comment:
I don't think you need a separate function for this. It should be a part of the fixture itself.